.gitattributes

```diff
@@ -0,0 +1,13 @@
+# Directories
+/.wordpress-org export-ignore
+/.github export-ignore
+
+# Currently only used for linting.
+/vendor export-ignore
+
+# Files
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.phpcs.xml export-ignore
```

Comment:
This file ensures that the SVN commit does not pass over development files:

https://github.com/10up/action-wordpress-plugin-deploy?tab=readme-ov-file#sample-baseline-files